.github/workflows/ci.yaml

```diff
@@ -24,3 +24,23 @@ jobs:
       - run: rm -rf node_modules
       - run: npm install
       - run: npm run test-generate
+  unit-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
```

Comment:
I'm glad you got initial testing in place 🎉 

Would like to work with @chingor13 to make our test runner smart about selectively running against only folders that have changed, as follow up work.